_studio/mfx_lib/shared/src/mfx_h264_fei_vaapi.cpp

```diff
@@ -693,113 +691,83 @@ mfxStatus VAAPIFEIPREENCEncoder::QueryStatus(
         vaSts = VA_STATUS_SUCCESS;
     MFX_CHECK_WITH_ASSERT(VA_STATUS_SUCCESS == vaSts, MFX_ERR_DEVICE_FAILED);
 
-    {
-        MFX_AUTO_LTRACE(MFX_TRACE_LEVEL_EXTCALL, "vaQuerySurfaceStatus");
-        vaSts = vaQuerySurfaceStatus(m_vaDisplay, waitSurface, &surfSts);
-    }
-    MFX_CHECK_WITH_ASSERT(VA_STATUS_SUCCESS == vaSts, MFX_ERR_DEVICE_FAILED);
+    VAMotionVector* mvs;
```

Comment:
These 2 defs somewhat hang in the air. Can you move them under corresponding ifs below (I don't see you use them on this scope level)?

Reply:
done